crates/swc_ecma_minifier/src/metadata/tests.rs

```diff
@@ -158,3 +158,63 @@
 // fn export_default_fn_1() {
 //     assert_standalone("export default function f(module, exports) {}", 0);
 // }
+
+use rustc_hash::FxHashSet;
+use swc_common::{Mark, DUMMY_SP};
+use swc_ecma_utils::{member_expr, quote_expr};
+
+use super::HashEqIgnoreSpanExprRef;
+
+#[test]
+fn test_hash_eq_ignore_span_expr_ref() {
+    use swc_ecma_ast::*;
+
+    fn expr_ref(expr_ref: &Expr) -> HashEqIgnoreSpanExprRef {
+        HashEqIgnoreSpanExprRef(expr_ref)
+    }
+
+    testing::run_test(false, |_cm, _handler| {
+        let dummy_sp = DUMMY_SP;
+        let meaningful_sp = dummy_sp.apply_mark(Mark::new());
+
```

Comment:
Can we have some check for expressions which is not an ident nor a member expression?

Reply:
Done